haskell.nix/library/.github/workflows/check-outdated.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  schedule:
+    - cron: '0 10 * * 1'
```

Comment:
As a person not used to `cron` I usually ask people to write a human-readable description near this syntax (in this case "At 10:00 on Monday" AFAIU).